### ISSUE TITLE
Search result ordering doesn't work on docket_id RECAP queries

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#842]
When you do a query for just one docket, such as:

    docket_id:22

You are shown the top 500 items from just that docket. That's great, but ordering doesn't work because normally ordering only affects the top-level group, not the items in the group. When we're doing a docket query, it should affect both instead since there's only one docket that will come back.

**Comment by mlissner** [comment]
I took a look at this today, but couldn't sort it out. Seemed easy, but then didn't work. There's a sorting parameter in the grouping, which is easy enough to tweak, and there's a sorting parameter in the template that is easy to remove, but even with both of them tweaked, the change didn't take. 

The parameters going to Solr seem fine, so it sort of seems like somewhere in the Python something is doing sorting that shouldn't be. That's....weird, but OK. This should be a fairly easy one for somebody to poke through and sort out, but I've got other projects on deck today.
